### PR TITLE
Adds glowy texture to gnesis material

### PR DESCRIPTION
**Pull request by TobleroneSwordfish**
[FEATURE] [BUG]
## About the PR 
Adds a glowing overlay to the gnesis material, and fixes many flock things having their appearance modified by being gnesis despite already being teal and flocky.
Pictured: a gnesis slag shovel and a gnesis and gnesisglass briefcase.
image

## Why's this needed? 
Pretty, and the flock textures being double tinted was bugging me.

### CODE REVIEW

**Comment by FlameArrow57** [comment]
I think how all the other stuff was done was using mat_changeappearance = FALSE or something, not sure if there's a difference

File: code/obj/flock/structure/flock_structure_parent.dm [hunk 1]
```diff
@@ -43,7 +43,7 @@ ABSTRACT_TYPE(/obj/flock_structure)
 	last_process = TIME
 	health_max = health
 	time_started = world.timeofday
-	setMaterial(getMaterial("gnesis"))
+	setMaterial(getMaterial("gnesis"), appearance = FALSE, setname = FALSE)
```
Comment (FlameArrow57) [inline code/obj/flock/structure/flock_structure_parent.dm:46]:
```suggestion
	setMaterial(getMaterial("gnesis"), appearance = FALSE)
```
Already has mat_changename = FALSE above?

File: code/obj/window.dm [hunk 2]
```diff
@@ -1046,11 +1046,11 @@
 // flock windows
 
 /obj/window/auto/feather
-	default_material = "gnesisglass"
 
 /obj/window/auto/feather/New()
 	connects_to += /turf/simulated/wall/auto/feather
 	..()
+	setMaterial(getMaterial("gnesisglass"), appearance = FALSE, setname = FALSE)
```
Comment (FlameArrow57) [inline code/obj/window.dm:1053]: Though these windows were supposed to inherit from the regular windows?

Comment (TobleroneSwordfish) [reply code/obj/window.dm:1053]: they do?

Comment (FlameArrow57) [reply code/obj/window.dm:1053]:
```suggestion
```
Err yeah, I don't think this is needed at all, testing it seems to already have the gnesisglass material from window/feather()

**Comment by FlameArrow57** [comment]
Can also remove mat_changeappearance from flock_structure/cage (also appears for /obj/item/reagent_containers/gnesis but bit more unrelated), and actually looking again at flock stuff, might be better to say mat_appearances_to_ignore = list("gnesis")

File: code/obj/flock/structure/flock_structure_parent.dm [hunk 3]
```diff
@@ -43,7 +43,7 @@ ABSTRACT_TYPE(/obj/flock_structure)
 	last_process = TIME
 	health_max = health
 	time_started = world.timeofday
-	setMaterial(getMaterial("gnesis"))
+	setMaterial(getMaterial("gnesis"), appearance = FALSE)
```
Comment (FlameArrow57) [inline code/obj/flock/structure/flock_structure_parent.dm:46]: Other Flock stuff just uses mat_appearances_to_ignore = list("gnesis"), not sure if matters

Comment (TobleroneSwordfish) [reply code/obj/flock/structure/flock_structure_parent.dm:46]: may as well be consistent 👍

**Pull request merged**